sharedresource/v1alpha1/0000_10_sharedresource.crd.yaml

```diff
@@ -0,0 +1,112 @@
+# this is the boilerplate crd def that controller-gen reads and modifies with the
+# contents from share_type.go
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: shares.sharedresource.openshift.io
+  annotations:
+    api-approved.openshift.io: https://github.com/openshift/api/pull/979
+    displayName: SharedResource
```

Comment:
@adambkaplan Is this ok for the name?

Reply:
Is `displayName` a required annotation?

I'd think `Share` would be the appropriate display name for the console. If so, we may not need to provide this annotation.